diff --git a/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js b/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js
index ff861a4..cf9df24 100644
--- a/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js
+++ b/node_modules/pdf-parse/lib/pdf.js/v2.0.550/build/pdf.js
@@ -15213,7 +15213,7 @@ var SVGGraphics = function SVGGraphics() {
         if (parseInt(process.versions.node) >= 8) {
           input = literals;
         } else {
-          input = new Buffer(literals);
+          input = Buffer.from(literals);
         }
         var output = require('zlib').deflateSync(input, { level: 9 });
         return output instanceof Uint8Array ? output : new Uint8Array(output);
